packages/web/tailwind-utilities.js

```diff
@@ -40,18 +40,18 @@ function insetUtilities({ matchUtilities, theme }) {
   matchUtilities(
     {
       start: (value) => ({
-        '[dir="rtl"] &': {
+        '[dir="rtl"] &, &[dir="rtl"]': {
```

Comment:
What are these for?

Reply:
Without that the dir attribute would have to be set on an ancestor node for the style to applied. This allows it to be set on the node itself as well.